### ISSUE TITLE
Outcomes of atomic tests

### ISSUE THREAD

**Issue by annika-FTB** [w3c/wcag-act#235]
In the current approach atomic tests can have two types of outcome.

1. Outcomes that are used as input to composed rules and should not be interpreted as "The content passes or fails an accessibility reuqirement."

_[At least this is how I understand the last paragrah of section 7 "Outcomes from an ACT Rule SHOULD be consistent with the accessibility requirement, ..." - which is a bit confusing by the way.]_

2. Outcomes that need no further aggregation and can be used directly to make a statement about an accessibility reuqirements.

3. Maybe there are even atomic rules which can be both?

In my opinion this distinction should be made more explicit. This could be done by restricting the use of the terms "pass" and "fail" to the second type of outcome. Or by adding composed rules as "wrapper" to all atomic rule, i.e. adding composed rules even if only one atomic rule is involved. Or by adding this information to the rule structure (see also #232).

This issue is also related to your question in Section 11: In my opinion "test definition" and "aggregation definition" should not be merged.

**Comment by annethyme** [comment]
Good point. 
I think we need a MUST and a note stating that atomic rules that are not consistent with the accessibility requirement on their own, must not list the accessibility requirement itself, but only to the composed rules they are part of

For ease of use, it would also be nice to refer to section 16 here from section 7 for how passes work.

**Comment by WilcoFiers** [comment]
I think I've got a solution to this in #273. I've created a distinction between rule results and SC outcomes. Rules can be passed / failed / inapplicable, where as success criteria can be satisfied.
